[CTOR-617] refresh OpenVPN OMI doc page (#3362)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/applications-openvpn-omi.md

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

## Pack assets

### Templates

The Monitoring Connector **OpenVPN OMI** brings a host template:

* **App-Openvpn-OMI-custom**

The connector brings the following service templates (sorted by the host template they are attached to):

<Tabs groupId="sync">
<TabItem value="App-Openvpn-OMI-custom" label="App-Openvpn-OMI-custom">

| Service Alias | Service Template                    | Service Description |
|:--------------|:------------------------------------|:--------------------|

> | Server-Usage  | App-Openvpn-Server-Usage-OMI-custom | Check server usage  |

Reviewer: ```suggestion
| Server-Usage  | App-Openvpn-Server-Usage-OMI-custom | Check VPN server usage  |
```

> The Open VPN server must have an management interface running and available. 

Reviewer: ```suggestion
The Open VPN server must have a management interface running and available. 
```

> A user and password dedicated to monitoring is needed for security reasons.

Reviewer: ```suggestion
A user and password dedicated to the monitoring are needed for security reasons.
```

```bash
yum install centreon-pack-applications-openvpn-omi
```

</TabItem>
</Tabs>

2. Whatever the license type (*online* or *offline*), install the **OpenVPN OMI** connector through
the **Configuration > Monitoring Connector Manager** menu.

### Plugin

Since Centreon 22.04, you can benefit from the 'Automatic plugin installation' feature.
When this feature is enabled, you can skip the installation part below.

You still have to manually install the plugin on the poller(s) when:
- Automatic plugin installation is turned off
- You want to run a discovery job from a poller that doesn't monitor any resource of this kind yet

> More information in the [Installing the plugin](/docs/monitoring/pluginpacks/#installing-the-plugin) section.

Use the commands below according to your operating system's package manager:

<Tabs groupId="sync">
<TabItem value="Alma / RHEL / Oracle Linux 8" label="Alma / RHEL / Oracle Linux 8">

```bash
dnf install centreon-plugin-Applications-Openvpn-Omi
```

</TabItem>
<TabItem value="Alma / RHEL / Oracle Linux 9" label="Alma / RHEL / Oracle Linux 9">

```bash
dnf install centreon-plugin-Applications-Openvpn-Omi
```

</TabItem>

> <TabItem value="Debian 11" label="Debian 11">

Reviewer: ```suggestion
<TabItem value="Debian 11 & 12" label="Debian 11 & 12">
```
